tests/test_carbon.py

```diff
@@ -36,7 +37,8 @@ def test_create_image_for_one_liner(python_kwargs):
     create_code_image(ONE_LINE_SNIPPET, **python_kwargs)
     assert CARBON_DOWNLOAD_FILE.exists()
     image_text = pytesseract.image_to_string(CARBON_DOWNLOAD_FILE.name)
-    assert "hello world" in image_text
+    # not sure why carbon.now.sh started to put stuff on a new line ...
```

Comment:
Other weirdness, not related to this PR but unfortunate:

<img width="1416" alt="Screenshot 2022-11-27 at 20 24 25" src="https://user-images.githubusercontent.com/387927/204156193-e900d041-69bf-4c8e-8590-136ca87187cc.png">

Seems an open issue: 
https://github.com/carbon-app/carbon/issues/743

Reply:
Logged in an issue, maybe good to try with Playwright ...